style.css

```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans:wght@200&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Oswald", sans-serif;
+}
+body {
+  height: 100vh;
+
+  display: grid;
+  place-items: center;
+
+  background: #3d3d3d;
+}
```

Comment:
It would be cool to avoid user selection on this 3D page.

Before:

![image](https://user-images.githubusercontent.com/1282684/229200935-95f963e6-172e-4401-a792-9092111dd5d7.png)


After

![image](https://user-images.githubusercontent.com/1282684/229201017-ecb751d1-41aa-4a76-8a33-8d2c3791a28e.png)


```suggestion
  user-select: none;
}
```

Reply:
Sure I'll try implementing it